### DISCUSSION TITLE
How to increase the size of a tooltip and the text within it.

### DISCUSSION THREAD

**Discussion by StormingBullet** [jgraph/drawio#3947] [Q&A]
I'm really enjoying this application, but I'm having a big problem with visibility.
I'm using this application for simple scenario writing that summaries on shape and details in Data (ctrl+m).
The problem is that the size of the tooltips and font size within them are too small.
Is there any way to change this size?

**Comment by lacikawiz** [comment]
I found a solution to this:
- Go to Extras -> Configuration in the menu.
- Enter the following code:
```json
{
  "css": ".geEditor div.mxTooltip {font-size: 20px}"
}
```
Then save and restart. 
You can change the font-size to your liking.

Also, if you need the dialog boxes bigger font then append another css rule:
```json
{
  "css": ".geEditor div.mxTooltip {font-size: 20px} .geDialog { transform : translate(-50%, -50%) scale(1.2) } "
}
```
Note: used the `scale` here because changing the font-size property didn't change the input boxes' font size, only the labels.

You can find these goodies if you poke around with the menu item: Help -> Open Developer Tools

**Reply by davjos99** [reply]
How do you make the dialog box of the tooltip constrain to a certain width?  Because right now if I have several paragraphs, the text spans the entire desktop and makes it a little hard to read.

**Reply by lacikawiz** [reply]
@davjos99 , You can use any CSS that you would use on a webpage. So you can add the `max-width` property which would limit the width of the dialog. You can look at the documentation on that at: 
https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Properties/max-width 

So, you can modify this part to something like this:

`div.mxTooltip {font-size: 20px}`   -->   `div.mxTooltip {font-size: 20px;max-width:80vw}`

> Note: `vw` is a unit equal to 1% of the viewport width (your window's width minus the scroll bar, if there's one). There are other units you can use, for example `px`, for pixel, etc. These you can find in CSS documentation pages.
